sky/server/common.py

```diff
@@ -293,7 +293,17 @@ def _start_api_server(deploy: bool = False,
                 time.sleep(0.5)
             else:
                 break
-        logger.info(ux_utils.finishing_message('SkyPilot API server started.'))
+
+        dashboard_msg = (
+            f'{colorama.Style.DIM}Dashboard: {get_server_url(host)}/dashboard.')
+        if not os.path.isdir(server_constants.DASHBOARD_DIR):
+            dashboard_msg += (
+                ' Please run `npm run build` to generate the dashboard if you'
+                ' install SkyPilot from source.')
```

Comment:
Should we avoid showing the dashboard URL if it is not working in this case?
Also, do we need to restart API server after the build? If yes, can we have a separate command to start the dashboard, so people don't need to restart the whole API server?

Reply:
Also, this line should be printed in a new line, instead of directly append to the API server URL, to make it better looking

Reply:
No need to restart API server after the build. Since no need to restart after the build, it should be OK to show the dashboard URL by default?
Here is the updated output:
```
sky api start
Failed to connect to SkyPilot API server at http://127.0.0.1:46580. Starting a local server.
✓ SkyPilot API server started. Dashboard: http://127.0.0.1:46580/dashboard
    Please run `npm run build` to generate the dashboard if you install SkyPilot from source.
├── SkyPilot API server: http://127.0.0.1:46580
└── View API server logs at: ~/.sky/api_server/server.log
```

Reply:
Can we add the prefix and make the build hint yellow?

Reply:
Updated